src/services/podcast.ts

```diff
@@ -6,7 +6,9 @@ import config from '~/config'
 const { API_BASE_URL } = config()
 
 export const getPodcastsByQuery = async (query) => {
-  const filteredQuery: any = {}
+  const filteredQuery: any = {
+    maxResults: true
```

Comment:
Can it affect other areas of the web app, since we now have no limit of results? Before it was 20.